src/Scan.js

```diff
@@ -1133,6 +1199,9 @@ export class Scan extends React.Component {
         {nextRequest && deliveryItem && this.renderDeliveryModal(deliveryItem, staffSlipContext)}
         {transitItem && this.renderTransitionModal(transitItem, staffSlipContext)}
         {itemError && this.renderErrorModal(itemError)}
+        {showActionChoiceModal &&
+         this.renderActionChoiceModal(items.records?.[0]?.items?.[0],
```

Comment:
1. Could you please clarify we need pass "this.props.stripes" to "renderActionChoiceModal" or we can just use "this.props.stripes" like we use " const { formatMessage } = this.props.intl;"?
2. Could you please clarify can we move "items.records?.[0]?.items?.[0]" to variable ?

Reply:
1. I think you're suggesting that `renderActionModal`, rather than its caller, could pull `servicePoint` out of the Stripes object. That's true. Would you like me to change it to do this?
2. Are you saying you would like me to assign `items.records?.[0]?.items?.[0]` to a temporary variable higher up the function, and pass that into `renderActionModal`?

I can make either or both of these changes if you wish.

Reply:
Hello, 
1. Yes 
2. Yes
Thank you